pkg/rosa/versions.go

```diff
@@ -98,3 +110,47 @@ func (c *RosaClient) getControlPlaneUpgradePolicies(clusterID string) (controlPl
 	}
 	return
 }
+
+// machinepools can be created with a minimal of two minor versions from the control plane.
+const minorVersionsAllowedDeviation = 2
+
+func MachinePoolSupportedVersionsRange(controlPlaneVersion string) (*semver.Version, *semver.Version, error) {
+	maxVersion, err := semver.Parse(controlPlaneVersion)
+	if err != nil {
+		return nil, nil, err
+	}
+
+	minVersion := semver.Version{
+		Major: maxVersion.Major,
+		Minor: max(0, maxVersion.Minor-minorVersionsAllowedDeviation),
+		Patch: 0,
+	}
+
+	if minVersion.LT(MinSupportedVersion) {
+		minVersion = MinSupportedVersion
+	}
+
+	return &minVersion, &maxVersion, nil
+}
+
+const versionPrefix = "openshift-v"
+
+// RawVersionID returns the rawID from the provided OCM version object.
+func RawVersionID(version *cmv1.Version) string {
```

Comment:
Please simply import [these](https://github.com/openshift/rosa/blob/8d35bae1dfe8a78c4f58c9d45f1467f1e853d449/pkg/ocm/versions.go#L238).

Reply:
I thought we agreed to not import the rosa cli repo, but work on creating a separate shared library?

Reply:
No. We are importing this.